### PR TITLE
chore: update package.json for remocal test

### PR DESCRIPTION
**Pull request by wojciechka**

### CODE REVIEW

**Comment by hoorayimhelping** [comment]
@wojciechka when will this get applied? The linked issue looks like it will go in next week?

**Comment by wojciechka** [comment]
> @wojciechka when will this get applied? The linked issue looks like it will go in next week?

Yes, I plan to merge this after all the other changes are merged. I'm just preparing all the PRs in advance.

**Review by drdelambre — APPROVED** [review]
good luck!

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**
